smack-core/src/main/java/org/jivesoftware/smack/ConnectionConfiguration.java

```diff
@@ -688,6 +702,21 @@ public B setResource(Resourcepart resource) {
             return getThis();
         }
 
+        /**
+         * Set the stream language.
+         * <p>
+         * (see RFC 6120 § 4.7.4).
+         * </p>
+         *
+         * @param language the language to use.
+         * @return a reference to this builder.
+         * @see <a href="https://tools.ietf.org/html/rfc6120#section-4.7.4">RFC 6120 § 4.7.4</a>
+         */
```

Comment:
I suggest to additionally add a `@see` to linking to "XML 1.0 § 2.12" https://www.w3.org/TR/xml/#sec-lang-tag